File: backend-springboot/src/main/java/com/javeriana/pontimovil/ponti_movil/dto/gestion_buses/bus_recibir/bBusRecibirDTO.java
package com.javeriana.pontimovil.ponti_movil.dto.gestion_buses.bus_recibir;

import lombok.*;
import java.io.Serializable;
import java.util.List;
import java.util.UUID;

@Getter
@Setter
@AllArgsConstructor
@EqualsAndHashCode
@NoArgsConstructor
public class bBusRecibirDTO implements Serializable {
    UUID id;
    String placa;
    String modelo;
    List<String> diasSemana;
    List<bRutaRecibirDTO> rutas;
    List<bConductorRecibirDTO> conductores;
}

File: frontend-angular/src/app/dto/gestionar-bus/bus-recibir/bBusRecibirDTO.ts
import {BCoductoresRecibirDTO} from './bCoductoresRecibirDTO';
import {BRutaEnvioDTO} from '../bus-envio/bRutaEnvioDTO';

export class BBusRecibirDTO {
  constructor(
    public id: string,
    public placa: string,
    public modelo: string,
    public diasSemana: string[],
    public rutas: BRutaEnvioDTO[],
    public conductores: BCoductoresRecibirDTO[]
  ) {}
}
